.cirrus.yml

```
@@ -129,7 +129,7 @@ cross_build_task:
 
```

Given the error message in the log for the smoke test, I'd suggest
```suggestion

    env:
        PATH: "/usr/local/opt/go@1.18/bin:$PATH"

```